### PR TITLE
Add support for Ory auth

### PR DESCRIPTION
**Pull request by mpgxvii**
- Adds support for Ory auth in front-end
- When creating a `RestSourceUser` and it already exists, return the existing user (needed when for external authorisation e.g. SEP user)
- Create custom `ManagementPortalResourceEnhancer` to bind custom MPClient
- Also created custom `MPClientFactory` to update client credentials config (to work with Ory hydra)
- Depends on https://github.com/RADAR-base/radar-commons/pull/176

### CODE REVIEW

File: authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/MPClient.kt [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        HttpClient(CIO) {
+            install(ContentNegotiation) {
+                json(Json { ignoreUnknownKeys = true })
+            }
+            install(HttpTimeout) {
+                requestTimeoutMillis = 30_000
+            }
+        }
+
+    private var accessToken: String? = null
```
Comment (yatharthranjan) [inline authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/MPClient.kt:45]: use lateinit

File: buildSrc/src/main/kotlin/Versions.kt [hunk 2]
```diff
@@ -6,7 +6,7 @@ object Versions {
 
     const val kotlin = "1.9.23"
 
-    const val radarCommons = "1.1.2"
+    const val radarCommons = "1.1.3-SNAPSHOT"
```
Comment (mpgxvii) [inline buildSrc/src/main/kotlin/Versions.kt:9]: Will update when released.

**Review by yatharthranjan — APPROVED** [review]
Looks great -- reusing the jersey code. Thanks!

**Review by yatharthranjan — APPROVED** [review]
LGTM

**Pull request merged**